### DISCUSSION TITLE
Text set not able to display any information

### DISCUSSION THREAD

**Discussion by dadoef** [ExperienceLovelace/ha-floorplan#379] [Q&A] [answered]
I started with my floorplan card in Lovelace. I got to the state that my motion sensors are changing the room color but I'm unable to get any data filled with the text_set service.
My code looks like this: 
`          - element: temp-bureau
              entity: sensor.t_h_sensor_bureau_temperatuur
              state-action:
                action: call-service
                service: floorplan.text_set
                service_data: ${entity.state}
            - element: humid-bureau
              entity: sensor.t_h_sensor_bureau_luchtvochtigheid
              state-action:
                action: call-service
                service: floorplan.text_set
                service_data: ${entity.state}
         - element: temp-bureau
              entity: sensor.t_h_sensor_bureau_temperatuur
              state-action:
                action: call-service
                service: floorplan.text_set
                service_data: ${entity.state}
            - element: humid-bureau
              entity: sensor.t_h_sensor_bureau_luchtvochtigheid
              state-action:
                action: call-service
                service: floorplan.text_set
                service_data: ${entity.state}`

When I hover over the text in the floorplan card I can see the status of the entity but for some reason the text keeps showing N/A and does not update to said status.
<img width="162" alt="Screenshot 2023-11-23 192112" src="https://github.com/ExperienceLovelace/ha-floorplan/assets/1028288/75768003-d9d4-472b-83a8-d68ff6b3eb73">
The same also is happening when I try to display the state of my Media Player.

Any ideas what could be going wrong?

**Comment by exetico** [comment]
Hi

Please format your code.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Also, share your SVG and YAML as a file, if possible. Provide screenshots too, also from the state. I bet you're missed something.

Take a look at the following guides:
https://www.youtube.com/watch?v=5ue9J4WG2a8&list=PL5xKVw-BInX1phV-Tnjznwd2YG5mEOvWL

**Reply by dadoef** [reply]
Thanks @exetico for the tips. I hope it's fine now.

**Comment by dadoef** [comment, marked as answer]
Found the error: state_action
